js/scripts.js

```diff
@@ -353,7 +353,10 @@ $(window).load(function() {
         return pattern.test(emailAddress);
         };
 
-	$('.email-subscribe').submit(function() {
+	$('.email-subscribe').submit(function(data) {
```

Comment:
Sendy will already check if the Honeypot field is filled or not. I don't think you need to check as well.